src/Microsoft.DotNet.XHarness.CLI/Commands/WASM/Browser/WasmBrowserTestRunner.cs

```diff
@@ -275,9 +276,32 @@ private string BuildUrl(ServerURLs serverURLs)
                 sb.Append('&');
 
             sb.Append($"arg={HttpUtility.UrlEncode(arg)}");
-        }
+        }
+
+        if (sb.Length > 0)
+            sb.Append('&');
+
+        sb.Append($"arg=-verbosity&arg={HttpUtility.UrlEncode($"{VerbosityToString()}")}");
 
         uriBuilder.Query = sb.ToString();
         return uriBuilder.ToString();
-    }
+    }
+
+    //MinimumLogLevel.Critical,
+    //MinimumLogLevel.Error,
+    //MinimumLogLevel.Warning,
+    //MinimumLogLevel.Info,
+    //MinimumLogLevel.Debug,
+    //MinimumLogLevel.Verbose
```

Comment:
nit: space after `//` and sort the lines to be the same order as the switch below